docs/build/changelog.rst

```diff
@@ -6,6 +6,16 @@ Changelog
     :version: 1.1.9
     :include_notes_from: unreleased
 
+    .. change::
+        :tags: feature, region
```

Comment:
**mike bayer** (zzzeek) wrote:

oh, technicality...this should be in a text file by itself inside of docs/build/unreleased/233.rst    it gets merged here using tooling later on.   

for an example see a previous commit like https://github.com/sqlalchemy/dogpile.cache/commit/f450a785aa9156d86216b194b9bf64791120abd9#diff-1568e46bf5b517b20c8f1ac9e9c2ff79134ac2be7695dee4c6cfa7bc5b8b92e0

View this in Gerrit at https://gerrit.sqlalchemy.org/c/sqlalchemy/dogpile.cache/+/4370

Reply:
gotcha - will make the change